.github/workflows/checks.yml

```diff
@@ -32,7 +32,6 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install poetry
-          poetry config installer.modern-installation false
```

Comment:
note: This poetry installer config was added in https://github.com/software-mansion/starknet.py/pull/881/files, but it looks like we don't need it anymore, thanks to https://github.com/python-poetry/poetry/pull/9392

Below is part of [Poetry 2.0.0 changelog](https://github.com/python-poetry/poetry/releases/tag/2.0.0) 👇 
<img width="688" alt="image" src="https://github.com/user-attachments/assets/a284301d-26bf-41da-bab3-fefe584bc6d7" />